src/AzureDevOpsRestApi/Viewmodel/Extractor/KeyConfig.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
 
 namespace AzureDevOpsAPI.Viewmodel.Extractor
 {
     public class KeyConfig
     {
-        public class Keys
+        public class keys
```

Comment:
Any reason this is no longer capitalized? I though it was good practice to capitalize class names.

Reply:
Yes, you are right!!, It should have been capitalize 🔠  
There was a problem with deserializing the keys, so I tried to lowercase this.

Later I tried the ```JsonProperty``` to resolve that - as you can see below. 😄 I'll correct this.